common/bitutil/doc.go

```diff
@@ -10,5 +10,41 @@
 //
 // TODO-Klaytn: put the original LICENSE file in the common/bitutil directory
 
-// Package bitutil implements fast bitwise operations.
+/*
+Package bitutil implements fast bitwise operations and compression/decompressions
+
+Bitwise Operations
+
+Following operations are supported
+  - AND, OR, XOR operations
+  - Provides both safe version and fast version of above operations
+    `Safe` means it can be performed on all architectures
+    `Fast` means it only can be performed on architecture which supports unaligned read/write
+
+Compression and Decompression
+
+Following operations are supported
+  - CompressBytes
+  - DecompressBytes
+
+How compression works
+
+The compression algorithm implemented by CompressBytes and DecompressBytes is
+optimized for sparse input data which contains a lot of zero bytes. Decompression
+requires knowledge of the decompressed data length.
+
+	Compression works as follows:
+
+	  if data only contains zeroes,
+		  CompressBytes(data) == nil
+	  otherwise if len(data) <= 1,
```

Comment:
what about `else if` instead of `otherwise if`

Reply:
@laggu Actually it was copied from the code comments. =)